operations/update.go

```diff
@@ -183,14 +170,14 @@ func prepareUpdate(url, newVersion string) (string, error) {
 	cmd := exec.Command(tempPath, "version")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return "", errors.Errorf("Update failed - checking version of new binary returned error: %v", err)
+		return "", errors.Errorf("Update failed - checking version of new binary returned error: %s", err)
```

Comment:
wraperr

Reply:
I disagree. I don't think the stack is useful here. The error is about what shelling out to a binary returned.

Reply:
I don't think we're printing the stack except in debugging/server side contexts. Wrapping just avoids weird string formatting